docs/spec/lcd/api.md

```diff
@@ -0,0 +1,847 @@
+
+Functionality: Create new seed
+
+Parameters: null
+
+* The above command returns JSON structured like this if success:
+
+```
+{
+    "error": "",
+    "code":200,
+    "result": {
+    	"seed":crime carpet recycle erase simple prepare moral dentist fee cause pitch trigger when velvet animal abandon
+    },
+    "rest api": "2.0"
+}
+
+```
+
+* The above command returns JSON structured like this if fails:
+
+```
+{
+    "error": "cannot generate new seed",
+    "code":500,
+    "result": {},
+    "rest api": "2.0"
+}
+        
+```
+
+
+### /keys/{name} - GET
+
+url: /keys/{name}, Method: GET
+
+Functionality: Get key information according to the specified key name
+
+Parameters: null
+
+* The above command returns JSON structured like this if success:
+
+```
+{
+    "error": "",
+    "code":200,
+    "result": {
+    	"name": "test",
+          "address": "cosmosaccaddr1thlqhjqw78zvcy0ua4ldj9gnazqzavyw4eske2",
+          "pub_key": "cosmosaccpub1zcjduc3qyx6hlf825jcnj39adpkaxjer95q7yvy25yhfj3dmqy2ctev0rxmse9cuak"
+    },
+    "rest api": "2.0"
+}
+
+```
+
+* The above command returns JSON structured like this if fails:
+
+```
+{
+    "error": "cannot find corresponding name",
+    "code":500,
+    "result": {},
+    "rest api": "2.0"
+}
+```
+
+
+### /keys/{name} - PUT
+
+url: /keys/{name}, Method: PUT
+
+Functionality: Update key password
+
+Parameters: 
+
+| Parameter       | Type   | Default | Required | Description     |
+| --------------- | ------ | ------- | -------- | --------------- |
+| old_password    | string | null    | true     | password before |
+| new_password    | string | null    | true     | password before |
+| repeat_password | string | null    | true     | password before |
+
+* The above command returns JSON structured like this if success:
+
+```
+{
+    "error": "",
+    "code":200,
+    "result": {
+     "updated":name
+    },
+    "rest api": "2.0"
+}
+```
+
+* The above command returns JSON structured like this if fails:
+
+```
+{
+    "error": "cannot update the corresponding key",
+    "code":500,
+    "result": {},
+    "rest api": "2.0"
+}
+```
+
+
+### /keys/{name} - DELETE
+
+url: /keys/{name}, Method: DELETE
+
+Functionality: Delete key from keystore
+
+Parameters:
+
+| Parameter | Type   | Default | Required | Description      |
+| --------- | ------ | ------- | -------- | ---------------- |
+| password  | string | null    | true     | password of keys |
+
+* The above command returns JSON structured like this if success:
+
+```
+{
+    "error": "",
+    "code":200,
+    "result": {
+     "deleted":name
+    },
+    "rest api": "2.0"
+}
+```
+
+* The above command returns JSON structured like this if fails:
+
+```
+{
+    "error": "cannot delete the corresponding key",
+    "code":500,
+    "result": {},
+    "rest api": "2.0"
+}
+```
+
+
+## ICS20 - TokenAPI
+
+The TokenAPI exposes all functionality needed to query account balances and send transactions.
+
+### /balance/{account} - GET
```

Comment:
Our accounts already hold the balance.
So why not have `/accounts/{account}` or `/accounts/{account}/tokens`?